transport/src/commonTest/kotlin/com/genesys/cloud/messenger/transport/shyrka/receive/FakeDeploymentConfig.kt

```diff
@@ -1,26 +1,23 @@
 package com.genesys.cloud.messenger.transport.shyrka.receive
 
-fun fakeDeploymentConfig() = DeploymentConfig(
+fun testDeploymentConfig(
```

Comment:
For these kinds of methods which are basically factory methods, I was wondering if it would make sense to prefix them with "create" or "make". My reasoning is that the word "test" can be a verb, so `testDeploymentConfig()` at first was ambiguous to me in knowing if it was performing some kind of test and then returning a `DeploymentConfig` or just creating a `DeploymentConfig`.